frontend/src/Settings/Quality/Definition/QualityDefinition.js

```diff
@@ -108,20 +111,20 @@ class QualityDefinition extends Component {
 
           <div className={styles.sizes}>
             <div>
-              <Label kind={kinds.WARNING}>{minThirty}</Label>
-              <Label kind={kinds.INFO}>{minSixty}</Label>
+              <Label kind={kinds.WARNING}>{minTwenty}</Label>
+              <Label kind={kinds.INFO}>{minFortyFive}</Label>
```

Comment:
Been meaning to add tool tips to these as well to let user know what they mean, want to go ahead and stick them in. You should be able to use the `title=` property of the label control.
Sonarr's old tips read `Maximum size for a 60 minute episode` etc..